### PR TITLE
Correcting case in documentation

### PR DESCRIPTION
**Pull request by amalkurup89**
**What this PR does / why we need it**: Correcting case in documentation

**Which issue(s) this PR fixes**: Documentation fix

### CODE REVIEW

File: docs/sources/administration/provisioning.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Currently we do not provide any scripts/manifests for configuring Grafana. Rathe
...
+It's possible to manage data sources in Grafana by adding one or more YAML config files in the [`provisioning/datasources`](/administration/configuration/#provisioning) directory. Each config file can contain a list of `datasources` that will be added or updated during start up. If the data source already exists, then Grafana updates it to match the configuration file. The config file can also contain a list of data sources that should be deleted. That list is called `deleteDatasources`. Grafana will delete data sources listed in `deleteDatasources` before inserting/updating those in the `datasource` list.
```
Comment (achatterjee-grafana) [inline docs/sources/administration/provisioning.md:68]: How about "that will get added or updated during startup". Otherwise looks good.

**Review by achatterjee-grafana — APPROVED** [review]
Please make the suggested change. 
Thank you!

File: docs/sources/administration/provisioning.md (hunk 1 above)
Comment (amalkurup89) [reply docs/sources/administration/provisioning.md:68]: @achatterjee-grafana Looks good for me

Comment (achatterjee-grafana) [reply docs/sources/administration/provisioning.md:68]: Great! Please make the change. PR is already approved. Thanks!

Comment (amalkurup89) [reply docs/sources/administration/provisioning.md]: @achatterjee-grafana Commited the suggested changes

Comment (achatterjee-grafana) [reply docs/sources/administration/provisioning.md]: @amalkurup89, it is ready for you to merge. Thanks!

**Pull request merged**